### PR TITLE
Add is-lb config param

### PR DESCRIPTION
**Pull request by davi17g**

### CODE REVIEW

File: basic-search/requirements.txt [hunk 1]
```diff
@@ -1 +1 @@
-aerospike-proximus==0.3.1
\ No newline at end of file
+aerospike-proximus==0.3.2.dev1
```
Comment (jdogmcsteezy) [inline basic-search/requirements.txt:1]: Should we wait until 0.3.2 is released to merge this PR?

**Review by jdogmcsteezy — CHANGES_REQUESTED** [review]

File: quote-semantic-search/quote-search/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # TODO: Include exact versions
 # Aerospike dependencies
-aerospike-proximus==0.3.1
+aerospike-proximus==0.3.2.dev1
```
Comment (hev) [inline quote-semantic-search/quote-search/requirements.txt:3]: Overall this seems good but we need to wait for 0.4.0 to be available before we merge this into main.

**Review by hev — CHANGES_REQUESTED** [review]

**Comment by arrowplum** [comment]
Can we merge this now?

**Comment by arrowplum** [comment]
https://pypi.org/project/aerospike-vector/0.4.0/

File: basic-search/requirements.txt (hunk 1 above)
Comment (arrowplum) [inline basic-search/requirements.txt:1]:
```suggestion
aerospike-proximus==0.4.0
```

File: prism-image-search/prism/requirements.txt [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # TODO: Include exact versions
 # Aerospike dependencies
-aerospike-proximus==0.3.1
+aerospike-proximus==0.3.2.dev1
```
Comment (arrowplum) [inline prism-image-search/prism/requirements.txt:3]:
```suggestion
aerospike-proximus==0.4.0
```

File: quote-semantic-search/quote-search/requirements.txt (hunk 2 above)
Comment (arrowplum) [inline quote-semantic-search/quote-search/requirements.txt:3]:
```suggestion
aerospike-proximus==0.4.0
```

**Comment by arrowplum** [comment]
We need to update the readmes config section

**Comment by jdogmcsteezy** [comment]
I have merged this branch into my dev branch VEC-132-0.4.0. I will open a new PR soon

**Pull request closed without merging**